activities_sticky_queues/worker.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+import logging  # noqa
+import random
+from typing import List
+from uuid import UUID
+
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+from activities_sticky_queues import tasks
+
+interrupt_event = asyncio.Event()
+
+
+async def main():
+    # Uncomment the line below to see logging
+    # logging.basicConfig(level=logging.INFO)
+
+    # Comment line to see non-deterministic functionality
+    random.seed(667)
+
+    # Create random task queues and create parameterised workflow
+    task_queues: List[str] = [str(UUID(int=random.getrandbits(128))) for _ in range(5)]
```

Comment:
```suggestion
    task_queues = [f"activity_sticky_queue-host-{UUID(int=random.getrandbits(128))}" for _ in range(5)]
```

Just to keep that prefix